src/items.js

```diff
@@ -78,7 +83,10 @@ const appCreator = (components) => {
         },
       });
       return {
-        body: tokenArea,
+        body: {
+          tokenArea,
+          keyDetails,
+        },
```

Comment:
Adding to the comment in components, I believe the body should already be built and pass down to components in its final form.

Reply:
Fixed ✅